content/integrate-linkedin-social-login-website/index.md

```diff
@@ -3,45 +3,45 @@ title: "Integrating LinkedIn Social Login on a Website"
 date: "2015-02-11"
 coverImage: "linkedin-feat-img.png"
 author: "Lucius Yu"
-tags: ["LinkedIn", "SocialLogin", "JavaScript"]
+tags: ["LinkedIn", "SocialLogin"]
 ---
 
-It is important nowadays to allow your user to sign into your website with their social network account, it provides better experience for your user and also Let's you obtain more information about your user. The most welcomed social platforms are Facebook, Twitter, LinkedIn and Google. In this article we will cover how to integrate LinkedIn social login authentication into your website.
+It is important nowadays to allow your user to sign in to your website with their social network account. It provides a better experience for your user and also lets you obtain more information about your user. The most welcomed social platforms are Facebook, Twitter, LinkedIn and Google. In this article, we will cover how to integrate LinkedIn social login authentication into your website.
 
 **Create a LinkedIn App**
```

Comment:
Use heading tags instead of bolding throughout the content as applicable where headings are there